CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+- Add nix::sys::posix_fallocate
```

Comment:
This should be under the "Added" section and rephrased like "Added `nix::sys::posix_fallocate`". Please also list the platforms that it was added for since it wasn't all platforms `nix` supports.